translations/en.json

```diff
@@ -78,9 +78,9 @@
   "contact_us": "Contact <1>Us</1>",
   "donate_subtitle": "Support Lemmy development",
   "donate_desc": "We are able to develop Lemmy as an open source platform free of tracking and ads thanks to the generosity of our users. Once a year we ask you to consider donating to support our work. Financial security allows us to continue maintaining and improving the platform. If you’d like to make a one-time or recurring donation simply click the button below. Thank you for using Lemmy.<br/><br/>Nutomic and Dessalines, Lemmy Developers",
-  "support_on_liberapay": "Support on <1>Liberapay</1>",
-  "support_on_patreon": "Support on <1>Patreon</1>",
-  "support_on_opencollective": "Support on <1>OpenCollective</1>",
+  "support_on_liberapay": "<1>Liberapay</1>",
```

Comment:
If this is the only link in the string, then you don't need to use html / interpolation. You only need these if you need to insert formatting in the middle of a sentence / paragraph.

Just do 

`<a href=... >{I18N.t("support_on_liberapay"))}</a>`